netci.groovy

```diff
@@ -69,7 +69,9 @@ class Constants {
                'gcstress0xc_minopts_heapverify1' : ['COMPlus_GCStress'  : '0xC', 'COMPlus_JITMinOpts'  : '1', 'COMPlus_HeapVerify'  : '1']
                ]
     // This is the basic set of scenarios
-    def static basicScenarios = ['default', 'pri1', 'ilrt', 'r2r', 'pri1r2r', 'gcstress15_pri1r2r', 'longgc', 'coverage', 'gcsimulator']
+    def static basicScenarios = ['default', 'pri1', 'r2r', 'pri1r2r']
+    // The following needs to be merged with above set once they longer need their own builds
+    def static basicScenarios = [ 'ilrt', 'gcstress15_pri1r2r', 'longgc', 'coverage', 'gcsimulator']
```

Comment:
Does groovy allow you to have two different variables with the same name? This seems like it might cause some confusion